src/Eloquent/Model.php

```diff
@@ -172,11 +174,7 @@ public function getAttribute($key)
         }
 
         // This checks for embedded relation support.
-        if (
-            method_exists($this, $key)
-            && ! method_exists(self::class, $key)
-            && ! $this->hasAttributeGetMutator($key)
```

Comment:
Not sure what this method does, but it seems the fix here is replace this with checking that the method returns either EmbedsOne or EmbedsMany (per your EmbedsRelations trait).